pom.xml

```diff
@@ -52,6 +52,13 @@
            <version>1.5</version>
            <optional>true</optional>
        </dependency>
+
+      <dependency>
+         <groupId>org.testng</groupId>
+         <artifactId>testng</artifactId>
```

Comment:
And also, there is no need to change permissions of `org.infinispan.configuration.parsing.ConfigurationParser` file.